src/browser/api/window.js

```diff
@@ -1081,6 +1081,7 @@ Window.disableFrame = function(identity) {
     }
 
     browserWindow.setUserMovementEnabled(false);
+    setOptOnBrowserWin('frame', false, browserWindow);
```

Comment:
It's a little confusing, but the window option `frame` and `disableFrame/enableFrame` aren't referring to the same thing. `disableFrame/enableFrame` should be renamed to something like `setMovable/setUnmovable`. These api calls toggle whether the user can move the window via mouse.